src/ros_movesm.ls

```diff
@@ -48,7 +48,7 @@ CONTROL_CODE    = 00000000 00000000;
     :  F[2:msm_dready]=(OFF) ;
     :   ;
     :  !move to point ;
-    :J PR[2:pos cache] R[3:velperc cache]% CNT R[4:cnt cache] Skip,LBL[10]    ;
+    :J PR[2:pos cache] R[3:velperc cache]% CNT R[4:cnt cache] ACC66 Skip,LBL[10]    ;
```

Comment:
This is rather robot & application specific. I get the idea, but let's not do this by default.

Could I ask you to remove this?